
Error:
Traceback (most recent call last):
  File "C:\AVRL\online-judge-v\online_judge\codes\965a0012-6c0c-40d4-85ea-383555333c8f_code.py", line 13, in <module>
    num = int(input())
EOFError: EOF when reading a line
